### PR TITLE
Add page-types to remaining pages

### LINKED ISSUE #27620: Add page-types for the Learn Area
As part of openwebdocs/project#91, we want to adde page-types to the Learn Area.

I propose to add 6 different page types:

Page type | meaning
-- | --
learn-topic | Overview of a section, or a topic (that is, a collection of modules)
learn-module | Overview of a module (with prerequisites, list of guides, …)
learn-module-chapter | A part of a module (explanations) (Such a guide has a prerequisite table and an overview table)
learn-module-assessment | The assignment at the end of a module or a guide
learn-faq| Specific page about a question

The complete list of the 350+ pages of the LA and the proposed page-view attribution is [here](https://docs.google.com/spreadsheets/d/1NWk8rEmVgNrLLn4QVMwgQYjqZHaol9dx6EqPAc9AHbA/edit#gid=0).

cc/ @wbamberg. What do you think?

### PR DESCRIPTION
**Pull request by teoli2003**
While doing the page-type project, we forgot some pages here or there (or they got added right after an area was converted without having a `page-type` YAML key.

This PR adds the missing page types in the 50-or-so pages that didn't have one. After this PR (and the two others still open), all MDN pages should have a `page-type`. (A last check will be made before closing the issue).

Note: no new page type is introduced by this PR

This is part of openwebdocs/project#91 and will close #27620.

### CODE REVIEW

File: files/en-us/mdn/at_ten/history_of_mdn/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: The history of MDN
 slug: MDN/At_ten/History_of_MDN
+page-type: guide
```
Comment (estelle) [inline files/en-us/mdn/at_ten/history_of_mdn/index.md:4]: is "history" a guide?

File: files/en-us/mdn/kitchensink/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: The MDN Content Kitchensink
 slug: MDN/Kitchensink
+page-type: guide
```
Comment (estelle) [inline files/en-us/mdn/kitchensink/index.md:4]: we may want to not have a page-type on this page.

Comment (teoli2003) [reply files/en-us/mdn/kitchensink/index.md:4]: I may use "test" as a specific page-type here.

**Comment by Josh-Cena** [comment]
I suggest "meta" for the meta docs

File: files/en-us/mdn/at_ten/history_of_mdn/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/mdn/at_ten/history_of_mdn/index.md:4]: >is "history" a guide?

Good question, looks like it's the best option we have at the moment: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/Page_type_key#generic_page_types

File: files/en-us/mdn/mdn_product_advisory_board/index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: MDN Product Advisory Board
 slug: MDN/MDN_Product_Advisory_Board
+page-type: guide
```
Comment (bsmth) [inline files/en-us/mdn/mdn_product_advisory_board/index.md:4]: This is another one that surfaces a need to have a different page type in future, maybe. Fine for the purpose of this PR though 👍🏻

File: files/en-us/mdn/writing_guidelines/page_structures/index.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Page structures
 slug: MDN/Writing_guidelines/Page_structures
+page-type: landing-page
```
Comment (bsmth) [inline files/en-us/mdn/writing_guidelines/page_structures/index.md:4]: is it a landing page or a writing guide?

**Review by bsmth — APPROVED** [review]
I'm leaving my +1 pending some of the (non-blocking) comments - trusting your judgement there. Thanks @teoli2003 👍🏻

**Comment by bsmth** [comment]
> I suggest "meta" for the meta docs

That's a good suggestion

**Pull request merged**